build/Build.cs

```diff
@@ -0,0 +1,164 @@
+using System;
```

Comment:
I don't see AppVeyor specific targets here (e.g. upload of tests results, version action selection, etc). Are you planning to add it later?

Reply:
Yes, this is still in progress.